### PR TITLE
Navbar added

### PR DESCRIPTION
**Pull request by AndresEstebanGonzalez**
## Pull request type

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## If its a bug, what was the behavior?

-

Card Link: https://trello.com/c/VWi7Nm5h

## What is the new behavior?
Navbar added.
-

## Other information

-

## Preview

Paste your screenshots here! :)

### CODE REVIEW

**Comment by nathan-hyan** [comment]
imagen


Beware of the formatting for the PR

File: src/App.tsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
 import Typography from '@mui/material/Typography';
 
-import Form from './Components/Form';
+import Navbar from './Components/Navbar';
```
Comment (nathan-hyan) [inline src/App.tsx:3]: use absolute imports

File: src/Components/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        position='static'
+        sx={{
+          boxShadow: 'none',
+        }}
+      >
+        <Toolbar
+          sx={{
+            height: '3.5rem',
+            color: '#000',
+            backgroundColor: '#D9D9D9',
```
Comment (nathan-hyan) [inline src/Components/Navbar.tsx:19]: use the color constants from material iu

File: src/Components/Navbar.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+          }}
+        >
+          <Typography
+            variant='h6'
+            component='div'
```
Comment (nathan-hyan) [inline src/Components/Navbar.tsx:24]: if its a text, it can't be a div

File: src/Components/Navbar.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+      >
+        <Toolbar
+          sx={{
+            height: '3.5rem',
+            color: '#000',
+            backgroundColor: '#D9D9D9',
+          }}
+        >
+          <Typography
+            variant='h6'
```
Comment (nathan-hyan) [inline src/Components/Navbar.tsx:23]: If its the main title, why use h6 instead of h1?

File: src/styles/main.css [hunk 5]
```diff
@@ -0,0 +1,3 @@
+body {
+  margin: 0;
```
Comment (nathan-hyan) [inline src/styles/main.css:2]: Is it necessary?

**Review by nathan-hyan — CHANGES_REQUESTED** [review]
Please fix comments and check warnings

File: src/styles/main.css (hunk 5 above)
Comment (AndresEstebanGonzalez) [reply src/styles/main.css:2]: Screenshot 2024-01-02 at 12 38 02

I have to add it or else it gives me a default margin on the body.

**Review by nathan-hyan — APPROVED** [review]

**Pull request merged**